tests/testthat/test-target-valid.R

```diff
@@ -372,7 +372,7 @@ test_that("get/set/archive object targets", {
   res <- st_db$get(t$name)
   expect_equal(res[names(res) != "time"],
                dep[names(dep) != "time"])
-  expect_gt(as.numeric(dep$time), as.numeric(res$time))
+  expect_gte(as.numeric(dep$time), as.numeric(res$time))
```

Comment:
This could be due to millisecond/second accuracy of timing functions.  I have a feeling that windows/posix have different resolutions here.